test/verifiedTests/dataIntegration/testMOOMIN/testWriteMoominOutput.m

```diff
@@ -0,0 +1,49 @@
+% The COBRAToolbox: testWriteMoominOutput.m
+%
+% Purpose:
+%     - Tests the writeMoominOutput function meant for exporting outputs of moomin
+%
+% Author:
+%     - Original file: Taneli Pusa 09/2019
+
+global CBTDIR
+
+% initialize the test
+currentDir = pwd;
+fileDir = fileparts(which('testWriteMoominOutput.m'));
+cd(fileDir);
+testPath = pwd;
+
+% create a toy example
+model = struct;
+model.rxns = {'rxn1'; 'rxn2'; 'rxn3'};
+model.rxnNames = model.rxns;
+model.genes = {'g1'; 'g2'; 'g3'; 'g4'};
+model.rules = {'x(1) | x(2)'; ''; '((x(1) & x(2)) | (x(3) & x(4)))'};
+model.subSystems = {'ss1'; 'ss2'; ''};
+model.inputColours = [1; 0; -1];
+model.weights = [1; -1; 2];
+model.outputColours = [1, 1; -1, -2; 6, 0];
+model.leadingGenes = [1; 0; 3];
+model.frequency = [1; 1; 0.5];
+model.combined = [1; 6; 6];
+expression.GeneID = {'g1'; 'g2'; 'g3'; 'g4'};
+expression.PPDE = [0.9; 0.9; 0.1; 0.1];
+expression.FC = [1; -1; 0.1; -0.1];
+model.expression = expression;
+
+% test with different input variants
+fprintf(' -- Running testWriteMoominOutput ... ');
+writeMoominOutput(model, 'test.out');
```

Comment:
Could you add comparison files to actually check, that the written output is correct?
Like this, the following `writeMoominOutput` file would pass this test:
```
function writeMoominOutput(varargin)
end
```

Reply:
You mean .txt files? Of course, but I'm not sure what would be the proper way to do the comparison?

Reply:
Have a look here:
https://github.com/opencobra/cobratoolbox/blob/c72a7bd511f79ebc1b6b1e0421544b22b4a9335d/test/verifiedTests/analysis/testPrint/testPrintRxnFormula.m